pkg/firewall/client.go

```diff
@@ -104,6 +104,7 @@ func (c *Client) DeleteBastionFirewallRule(ctx context.Context, cluster *capg.GC
 
 	if gcperrors.IsNotFound(err) {
 		// pass thru, resource is already deleted
+		return nil
```

Comment:
wouldn't it return `nil` anyway on line 113?

Reply:
hmm maybe you're right, then I don't get why it's failing with the error and not ignoring it